WordPress/src/main/java/org/wordpress/android/ui/people/PeopleManagementActivity.java

```diff
@@ -15,23 +18,31 @@
 import org.wordpress.android.ui.ActivityLauncher;
 import org.wordpress.android.ui.accounts.BlogUtils;
 import org.wordpress.android.ui.people.utils.PeopleUtils;
+import org.wordpress.android.util.ToastUtils;
 
 import java.util.List;
 
-public class PeopleManagementActivity extends AppCompatActivity implements PeopleListFragment.OnPersonSelectedListener {
+public class PeopleManagementActivity extends AppCompatActivity
+        implements PeopleListFragment.OnPersonSelectedListener, PersonDetailFragment.OnChangeListener {
     private static final String KEY_PEOPLE_LIST_FRAGMENT = "people-list-fragment";
     private static final String KEY_PERSON_DETAIL_FRAGMENT = "person-detail-fragment";
 
+    private Person mSelectedPerson;
```

Comment:
Would you consider keeping this state variable inside the PersonDetailFragment and fetch it from there when needed? Cheers!

This will also probably take care of the rotation problem (where the remove button doesn't do anything after rotating).